src/index.js

```diff
@@ -16,10 +16,13 @@ import ru from 'react-intl/locale-data/ru';
 import { IntlProvider } from 'react-intl';
 import { addLocaleData } from 'react-intl';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { Logger } from './utils/helperFunctions';
 import store from './store';
 
 addLocaleData([...en, ...fr, ...es, ...de, ...ru]);
 
```

Comment:
Add some inline comments, that console.* API won't work in production. Also, add at the place of function definition to avoid any confusion.

Reply:
@akshatnitd made the changes